examples/todo/src/openapi-spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import {OpenAPIObject} from '@loopback/rest';
+import {TodoListApplication} from './application';
+
+/**
+ * Export the OpenAPI spec from the application
+ */
+async function exportOpenApiSpec(): Promise<OpenAPIObject> {
```

Comment:
I find this design weird. At one hand, this method is returning `OpenAPIObject` which leaves the impression that perhaps it can be called from other JS/TS modules. On the other hand, the implementation reads `process.argv[2]`, which is unlikely to contain a meaningful value when this function is called from another module.

Personally, I see little value in exporting this function as a public JS/TS API. It's easy to obtain application's OpenAPI spec programmatically via `app.boot()` and `app.getApiSpec()`, plus callers will need to carefully consider what ApplicationConfig to use (`process.env`-based approach may not be the right solution).

I am proposing to simplify `src/openapi-spec.ts` along the lines of `src/migrate.ts`.

```ts
async function exportOpenApiSpec(): Promise<void> {
 // ...
}

exportOpenApiSpec().catch(err => {
  console.error(/*...*/);
  process.exit(1);
});
```

Reply:
The function is not exported. I fixed the return type.